backend/db/mongo.go

```diff
@@ -39,6 +42,34 @@ func (c *Database) insertArticle(article Article) (string, error) {
 // 	return User{}, nil
 // }
 
+// fetch all Article from mongo
+func (c *Database) FindAllArticles() (int64, []Article, error) {
+	collection := c.database.Collection("articles")
+	cursor, err := collection.Find(context.TODO(), bson.D{})
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
We should return here rather than log.Fatal here as that would shutdown the server